assets/blocks/reader-registration/edit.js

```diff
@@ -228,7 +228,7 @@ export default function ReaderRegistrationEdit( {
 																) }
 															</span>
 															{ displayListDescription && (
-																<span className="newspack-registration__lists__description">
+																<span className="newspack-reader__lists__description">
 																	{ listConfig[ listId ]?.description }
```

Comment:
The styles in the editor are broken for the Registration block:

<img width="820" alt="image" src="https://user-images.githubusercontent.com/7383192/181187815-fc67c417-1aad-44d8-832e-7178e23ab0f5.png">